diff -up smstools3/src/Makefile.Makefiletabpatch smstools3/src/Makefile
--- smstools3/src/Makefile.Makefiletabpatch	2010-09-21 13:29:14.000000000 +0200
+++ smstools3/src/Makefile	2015-07-14 12:07:10.014194426 +0200
@@ -43,7 +43,7 @@ smsd: smsd.c extras.o locking.o cfgfile.
 
 ifneq (,$(findstring SOLARIS,$(CFLAGS)))
 ifeq (,$(findstring DISABLE_INET_SOCKET,$(CFLAGS)))
-override LFLAGS += -lsocket -lnsl
+	override LFLAGS += -lsocket -lnsl
 endif
 endif
 
diff -up smstools3/src/Makefile.Makefiletabpatch smstools3/src/Makefile
--- smstools3/src/Makefile.Makefiletabpatch	2015-07-14 12:10:58.419280531 +0200
+++ smstools3/src/Makefile	2010-09-21 13:29:14.000000000 +0200
@@ -43,7 +43,7 @@ smsd: smsd.c extras.o locking.o cfgfile.
 
 ifneq (,$(findstring SOLARIS,$(CFLAGS)))
 ifeq (,$(findstring DISABLE_INET_SOCKET,$(CFLAGS)))
-	override LFLAGS += -lsocket -lnsl
+override LFLAGS += -lsocket -lnsl
 endif
 endif
 
diff -up smstools3/src/Makefile.Makefiletabpatch smstools3/src/Makefile
--- smstools3/src/Makefile.Makefiletabpatch	2010-09-21 13:29:14.000000000 +0200
+++ smstools3/src/Makefile	2015-07-14 12:10:58.419280531 +0200
@@ -43,7 +43,7 @@ smsd: smsd.c extras.o locking.o cfgfile.
 
 ifneq (,$(findstring SOLARIS,$(CFLAGS)))
 ifeq (,$(findstring DISABLE_INET_SOCKET,$(CFLAGS)))
-override LFLAGS += -lsocket -lnsl
+	override LFLAGS += -lsocket -lnsl
 endif
 endif
 
